### PR TITLE
UI version 0.1 (+simon)

### PR DESCRIPTION
**Pull request by Salomon-MH**

### CODE REVIEW

**Comment by daniel-scholz** [comment]
supi dupi

File: static/style.css [hunk 1]
```diff
@@ -5,9 +5,10 @@
 }
 
 .content {
-	height: 100%;
+	position: absolute;
```
Comment (niklaskorz) [inline static/style.css]: Fix indentation. Please don't mix spaces and tabs, use either one.

File: static/style.css [hunk 2]
```diff
@@ -19,11 +20,26 @@
...
 }
 
+.contentReg {
+	position: absolute;
+	left: 35%;
+	/*height: 65%;*/
+	width: 30%;
+	margin: 4% 0 6% 0;
+	background-color: #F4F4F4;
+    padding: 0;
```
Comment (niklaskorz) [inline static/style.css]: Same as above and throughout the rest of the file, stay with one indentation type.

File: static/style.css (hunk 1 above)
Comment (Salomon-MH) [reply static/style.css]: 5a10b9a

File: static/style.css (hunk 2 above)
Comment (Salomon-MH) [reply static/style.css]: 5a10b9a

**Pull request merged**